### PR TITLE
feat(pci-private-registry): add api url modal and users credentials 

### PR DESCRIPTION
**Pull request by seven-amid**
ref: DTCORE-2503
ref: DTCORE-2504
ref: DTCORE-2507

| Question         | Answer
| ---------------- | ---
| Branch?          |  feat/pci-private-registry
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes/no
| Tickets          | DTCORE-2503, DTCORE-2504 and DTCORE-2507
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-private-registry/src/api/data/registry.ts [hunk 1]
```diff
@@ -65,5 +65,24 @@ export const deleteRegistry = async (projectId: string, registryId: string) => {
...
+  id: string;
+  password: string;
+  user: string;
+};
+
+export const getRegistryCredentials = async (
+  projectId: string,
+  registryId: string,
+): Promise<TRegistryCredentials> => {
+  const { data } = await v6.post(
```
Comment (sidlynx) [inline packages/manager/apps/pci-private-registry/src/api/data/registry.ts:83]: a get may be?

File: packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  OsdsClipboard,
+  OsdsModal,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useGetAllRegistries } from '@/api/hooks/useRegistry';
+
+export default function ApiUrlPage() {
+  const { t } = useTranslation();
```
Comment (sidlynx) [inline packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx:21]: does useTranslation without a param defaults to common?

**Review by frenautvh — APPROVED** [review]

**Review by sidlynx — CHANGES_REQUESTED** [review]

File: packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx (hunk 2 above)
Comment (seven-amid) [reply packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx:21]: yeah sure

File: packages/manager/apps/pci-private-registry/src/api/data/registry.ts (hunk 1 above)
Comment (seven-amid) [reply packages/manager/apps/pci-private-registry/src/api/data/registry.ts:83]: no it's a post, i've tried the get but it's dosen't work

**Review by sidlynx — APPROVED** [review]

**Review by kqesar — APPROVED** [review]

**Review by sidlynx — APPROVED** [review]

File: packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            >
+              {t('private_registry_harbor_api_url')}
+            </OsdsText>
+            <OsdsClipboard
+              className="mt-4"
+              aria-label="clipboard"
+              value={registry?.url}
+            >
+              <span slot="success-message">{t('common_clipboard_copied')}</span>
+            </OsdsClipboard>
```
Comment (rjamet-ovh) [inline packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx:59]: You can use the manager-component Clipboard, you won't need to add another translation key.

File: packages/manager/apps/pci-private-registry/src/pages/credentials/Credentials.page.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+          >
+            {t('private_registry_credentials_login')}
+          </OsdsText>
+          <OsdsClipboard
+            className="mt-2"
+            aria-label="clipboard"
+            value={credentials.user}
+          >
+            <span slot="success-message">{t('common_clipboard_copied')}</span>
+          </OsdsClipboard>
```
Comment (rjamet-ovh) [inline packages/manager/apps/pci-private-registry/src/pages/credentials/Credentials.page.tsx:103]: You can use the manager-component Clipboard here also.

File: packages/manager/apps/pci-private-registry/src/pages/credentials/Credentials.page.tsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+          >
+            {t('private_registry_credentials_password')}
+          </OsdsText>
+          <OsdsClipboard
+            className="mt-2"
+            aria-label="clipboard"
+            value={credentials.password}
+          >
+            <span slot="success-message">{t('common_clipboard_copied')}</span>
+          </OsdsClipboard>
```
Comment (rjamet-ovh) [inline packages/manager/apps/pci-private-registry/src/pages/credentials/Credentials.page.tsx:119]: You can use the manager-component Clipboard here also.

**Pull request merged**